src/middlewares/auth/authStrategies.js

```diff
@@ -1,12 +1,30 @@
-const { User, UserStatuses, UserRoles } = require('../../models');
+const { User, UserStatuses } = require('../../models');
+const { UsersService } = require('../../services');
 const bcrypt = require('bcryptjs');
+
 require('dotenv').config();
 
-const verifyCustom = async (req, done) => {
-  const user = await User.findOne({ email: req.body.email }).exec();
+const usersService = new UsersService();
+
+const checkUser = async (req) => {
+  const user = await User.findOne({ email: req.email }).exec();
   const status = await UserStatuses.findOne({ name: 'Active' }).exec();
 
-  if (!user || user.status.toString() !== status._id.toString()) {
+  if (!user) {
+    return false;
+  }
+
+  if (user.status.toString() !== status._id.toString()) {
```

Comment:
You can write here just one check: if(!user || user.status.toString() !== activeStatus._id.toString()) {return false}

Reply:
Fixed.